engine/src/main/java/org/hibernate/search/engine/cfg/spi/ConvertUtils.java

```diff
@@ -310,7 +310,7 @@ public static BigInteger convertBigInteger(Object value) {
 					.collect( Collectors.toList() );
 		}
 		else if ( value instanceof String ) {
-			return separatorPattern.splitAsStream( (String) value )
```

Comment:
This isn't consistent with the `instanceof` above, nor with the other `convert` methods we have in the same class. I'd be in favor of leaving this as-is.